app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comment_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] I was unable to create comments for a given post, this is because there is a difference in the variable name declared in line 5 and line 7 as highlighted above. Kindly address it. 👍